### PR TITLE
build: Add script to update package.json and commit for next release

### PR DESCRIPTION
**Pull request by ad1992**
We will try this script for the next few releases and once this works out well we can automate publish via gh action as well so the release will be fully automated.

### CODE REVIEW

File: scripts/release.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+const exec = util.promisify(require("child_process").exec);
+const updateReadme = require("./updateReadme");
+const updateChangelog = require("./updateChangelog");
+
+const excalidrawDir = `${__dirname}/../src/packages/excalidraw`;
+const excalidrawPackage = `${excalidrawDir}/package.json`;
+
+const updatePackageVersion = () => {
+  const pkg = require(excalidrawPackage);
+  pkg.version = nextVersion;
```
Comment (dwelle) [inline scripts/release.js:12]: I'd pass this as an arg for clarity instead of a global

File: scripts/release.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    process.exit(1);
+  }
+};
+
+const nextVersion = process.argv.slice(2)[0];
+if (!nextVersion) {
+  console.error("Pass the next version to release!");
+  process.exit(1);
+}
+release(nextVersion);
```
Comment (dwelle) [inline scripts/release.js:39]: same here, you're passing it, but using the global in the end

File: scripts/release.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+};
+
+const release = async () => {
+  try {
+    updateReadme();
+    await updateChangelog(nextVersion);
+    updatePackageVersion();
+    await exec(`git add -u`);
+    await exec(
+      `git commit -m "docs: release excalidraw@exclidraw@${nextVersion}  🎉"`,
```
Comment (dwelle) [inline scripts/release.js:24]:
```suggestion
      `git commit -m "docs: release @excalidraw/excalidraw@${nextVersion}  🎉"`,
```

**Review by dwelle — APPROVED** [review]

File: scripts/release.js (hunk 3 above)
Comment (ad1992) [reply scripts/release.js]: nice catch :)

**Pull request merged**